tox.ini

```diff
@@ -52,6 +52,13 @@ deps=
   flake8
 commands=flake8 --select=F,E9 {posargs:teuthology scripts}
 
+[testenv:flake8-py3]
```

Comment:
Do you really want that commit in this PR? I think it's from https://github.com/ceph/ceph/pull/32222

Reply:
Yes, I want it, as early as possible.